core/modules/tests.rs

```diff
@@ -504,35 +504,9 @@ fn test_json_module() {
 
   let module_map = runtime.module_map().clone();
 
-  let (mod_a, mod_b, mod_c) = {
+  let (mod_b, mod_c) = {
     let scope = &mut runtime.handle_scope();
-    let specifier_a = ascii_str!("file:///a.js");
     let specifier_b = ascii_str!("file:///b.js");
-    let mod_a = module_map
-      .new_es_module(
-        scope,
-        true,
-        specifier_a,
-        ascii_str!(
-          r#"
-          import jsonData from './c.json' assert {type: "json"};
```

Comment:
I think you can only replace `assert` with `with` and it will still work. Unless we already have a test that cover import attributes

Reply:
Yup. This test covered both styles previously -- removed the legacy syntax one only, leaving the new syntax one.